### PR TITLE
WD-7771 - update buttons in /robotics

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Updated buttons in "Bringing Ubuntu’s worldwide infrastructure to your robot" section in https://ubuntu-com-13390.demos.haus/robotics

## QA

- [demo link](https://ubuntu-com-13390.demos.haus/)
- [copy doc](https://docs.google.com/document/d/1JNFk2C0shH3DapKuJM8X0QfAiD1RzHvXoydn8P9mEMg/edit#heading=h.fdlzz3sruhkt)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the buttons and links on them are correct

## Issue / Card
[WD-7771](https://warthogs.atlassian.net/browse/WD-7771)

Fixes #

## Screenshots

[WD-7771]: https://warthogs.atlassian.net/browse/WD-7771?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/robotics/index.html [hunk 1]
```diff
@@ -235,7 +237,8 @@ <h3 class="p-heading--2">Snapcraft</h3>
         <p>Snapcraft is the command-line tool for packaging your software as a Snap. Snaps are containerised software packages designed for embedded applications, with increased performance compared to <a href="/engage/dockerandros">other technologies</a>. Snaps give you all the interfaces you need to bundle your dependencies in one package, while providing secure interfaces to access the host disc and privileged resources.</p>
 
         <p>
-          <a href="https://snapcraft.io/docs/robotics" class="p-button--positive">Learn more about Snapcraft</a>
+          <a class="p-button--positive js-invoke-modal" href="/robotics/docs"> Explore documentation</a>
```
Comment (lorumic) [inline templates/robotics/index.html:240]: It seems that the link is different here:
```suggestion
          <a class="p-button--positive js-invoke-modal" href="/robotics/docs/how-to-guides#heading--snapcraft"> Explore documentation</a>
```

File: templates/robotics/index.html [hunk 2]
```diff
@@ -245,7 +248,8 @@ <h3 class="p-heading--2">IoT App Store</h3>
         <p>Companies can create either a public or private marketplace, where applications can be managed by developers and contributors.</p>
 
         <p>
-          <a href="/internet-of-things/appstore" class="p-button--positive">Learn more about IoT App Stores</a>
+          <a class="p-button--positive js-invoke-modal" href="/robotics/docs">Explore documentation</a>
```
Comment (lorumic) [inline templates/robotics/index.html:251]: And here too:
```suggestion
          <a class="p-button--positive js-invoke-modal" href="/robotics/docs/explanation#heading--iot-app-store">Explore documentation</a>
```

File: templates/robotics/index.html [hunk 3]
```diff
@@ -256,7 +260,8 @@ <h3 class="p-heading--2">Ubuntu Core</h3>
         <p>Ubuntu Core uses the same kernel, libraries, and system software as classic Ubuntu, resulting in a smooth transition to production.</p>
 
         <p>
-          <a href="/core" class="p-button--positive">Learn more about Ubuntu Core</a>
+          <a class="p-button--positive js-invoke-modal" href="/robotics/docs">Explore documentation</a>
```
Comment (lorumic) [inline templates/robotics/index.html:263]: And here too:
```suggestion
          <a class="p-button--positive js-invoke-modal" href="/robotics/docs/how-to-guides#heading--ubuntu-core">Explore documentation</a>
```

**Comment by lizzochek** [comment]
@lorumic yes, you are correct about the links indeed, my bad. accepted your suggestions

File: templates/robotics/index.html [hunk 4]
```diff
@@ -226,7 +227,8 @@ <h3 class="p-heading--2">Ubuntu</h3>
         <p>Every aspect of the development process benefits from Ubuntu's responsiveness, ease of use, regular software updates, lightweight nature, and a high degree of security.</p>
 
         <p>
-          <a href="/desktop" class="p-button--positive">Learn more about Ubuntu</a>
+          <a class="p-button--positive js-invoke-modal" href="/robotics/docs"> Explore documentation</a>
```
Comment (lorumic) [inline templates/robotics/index.html:230]:
```suggestion
          <a class="p-button--positive" href="/robotics/docs"> Explore documentation</a>
```

**Review by lorumic — APPROVED** [review]

**Pull request merged**